testing/run

```diff
@@ -24,26 +24,35 @@ prechecks() {
             echo "ARTIFACT_DIR not set, cannot proceed without inside the image."
             false
         fi
-
-        export ARTIFACT_DIR="/tmp/ci-artifacts_$(date +%Y%m%d)"
+        # Even though this repo is not mainly used for osde2e, it was decided to 
```

Comment:
I think you can remove this block, and let 
```
#  testing/osde2e/gpu-addon.sh
export ARTIFACT_DIR=/test-run-results 
```
overwrite `$ARTIFACT_DIR` (as currently happening)

this causes this `echo` to be wrong, but I don't think it matters much
```
        echo "Using ARTIFACT_DIR=$ARTIFACT_DIR as default artifacts directory."
```